LongWarOfTheChosen/Config/XComLW_UI.ini

```diff
@@ -0,0 +1,16 @@
+[LW_Overhaul.UIResistanceManagement_LW]
+m_HeaderFontSize = 24				; Resistance overview header buttons
```

Comment:
I'm trying to move to a standard of all caps with underscores for config variables. It's inelegant, but consistent with much of the base game and helps easily identify in the code what's configurable and what's not. Please use the same convention here (without the "m_" prefix).

Clearly there are a lot of cases where config vars aren't all caps in the current code, but got to start somewhere!

Reply:
Also, does this make the text bigger than normal? It seemed like it when I tested it, but wasn't entirely sure.